app.py

```diff
@@ -104,6 +104,47 @@ def experience(index = None):
             "message": "New experience successfully created",
             "id": len(data['experience']) -1
             }), 201
+    if request.method == 'DELETE':
+        if index and str(index).isnumeric():
+                expId = int(index)
```

Comment:
can you pleas follow the python convention of [snake_case](https://en.wikipedia.org/wiki/Snake_case)? i.e. `exp_id` like on line 121

Reply:
Done